src/04-modality-specific-files/10-microscopy.md

```diff
@@ -54,12 +54,10 @@ Microscopy raw data MUST be stored in one of the following formats:
     (`.ome.tif` for standard TIFF files or `.ome.btf` for
     [BigTIFF](https://www.awaresystems.be/imaging/tiff/bigtiff.html) files)
 
-If different from PNG, TIFF or OME-TIFF, the original unprocessed data in the native format MAY be
-stored in the [`/sourcedata` directory](../02-common-principles.md#source-vs-raw-vs-derived-data).
+-   [NGFF/OME-ZARR](https://ngff.openmicroscopy.org/latest/) (`.ome.zarr` directories)
```

Comment:
There is a paragraph above (lines 34-40) that explains the choice of OME-TIFF. It may be beneficial to mention NGFF there as well.